.github/workflows/test_build_multi_platform.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and Test Anomaly detection
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  Build-ad-windows:
+    strategy:
+      matrix:
+        java: [8, 11, 14]
+    name: Build and Test Anomaly Detection Plugin on Windows
+    runs-on: windows-latest
+    steps:
+      - name: Setup Java ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Checkout Anomaly Detection
+        uses: actions/checkout@v2
+
+      - name: Build and Run Tests
+        run: |
+          ./gradlew build
+      - name: Publish to Maven Local
+        run: |
+          ./gradlew publishToMavenLocal
+      - name: Upload Coverage Report
+        uses: codecov/codecov-action@v1
+        with:
+          file: ./build/reports/jacoco/test/jacocoTestReport.xml
+          flags: plugin
+  Build-ad:
+    strategy:
+      matrix:
+        java: [8, 11, 14]
+        os: [ubuntu-latest]
```

Comment:
Should we run on macos-latest as well?

Reply:
Didn't originally add this as it wasn't part of what is needed for 1.3.7 and was worried about the deadline. However since window is passing, just add macos along with the needed test fix I made in the past on 2.x branch.